lib/util/rng.c

```diff
@@ -236,8 +219,8 @@ double rng_get_double( rng_type * rng ) {
 }
 
 int rng_get_int( rng_type * rng , int max_value ) {
-  rng_safe_cast( rng );
-  return rng->forward( rng->state ) % max_value;
+  int x = (int) util_dev_urandom_seed();
```

Comment:
Ehhhh - no no; the `rng_get_int( )` function should get *the next* integer in a deterministic fashion, from the stateful `rng_type` instance. The proposed changes will just get a "truly random" value - that is *not* what we want.